cfgov/sheerlike/templates.py

```diff
@@ -4,16 +4,9 @@
 
 
 def _convert_date(date, tz):
-    if date:
-        if isinstance(date, basestring):
-            date = parser.parse(date,
-                                default=datetime.datetime.today().replace(day=1))
-        if type(date) in [datetime.datetime, datetime.date]:
-            if isinstance(date, datetime.datetime):
-                pytzone = timezone(tz)
-                if date.tzinfo:
-                    date = date.replace(tzinfo=None)
-                date = pytzone.localize(date)
+    if date and isinstance(date, basestring):
+        date = parser.parse(date,
+                            default=datetime.datetime.today().replace(day=1))
     return date
 
```

Comment:
Sure, we could put that in there. I just removed it for now because we don't have that issue since django is now rendering these datetimes in the templates correctly and we currently don't have a way to handle different timezones. `when` works fine with the change. If you want to add a commit that changes `_convert_date()` to the above code then I'm cool with it.